app/models/subnet.rb

```diff
@@ -187,7 +187,14 @@ def self.import(proxy)
     ProxyAPI::DHCP.new(:url => proxy.url).subnets.map do |s|
       # do not import existing networks.
       attrs = { :network => s["network"], :mask => s["netmask"] }
+
       next if first(:conditions => attrs)
+      if s["options"]
+        attrs.merge!({:gateway => s["options"]["gateway"],
```

Comment:
but then you would need to remove those which AR does not support, imho a
whitelist is a better approch (as a nitpick could use hash slice instead).